### ISSUE TITLE
Swift 3 error with unicode characters

### ISSUE THREAD

**Issue by segura2010** [tidwall/SwiftWebSocket#64]
Hi! I am using your library to build an app for a NodeBB forum. All have been working fine until i updated it to use Swift 3. Now, I cant parse the JSON text messages that the server sends me.

An example of the messages (it isnt a complete message):

```
{"topics":[{"tid":24909,"uid":400,"cid":"3","mainPid":790889,"title":"Retrasados just being retrasados una vez mOptional("\u{00E1}")s en Galicia","slug":"24909/retrasados-just-being-retrasados-una-vez-mOptional("\u{00E1}")s-en-galicia","timestamp":1473334344094,"lastposttime":1473414364688,"postcount":23,"viewcount":95,"locked":false,"deleted":false,"pinned":false,"teaserPid":"791474","titleRaw":"Retrasados just being retrasados una vez mOptional("\u{00E1}")s en Galicia"}]}
```

As you can see, there are multiple Optional("\u{????}") in the text message. When I try to use  JSONSerialization.jsonObject to parse the JSON message, it gives me the error:

Error Domain=NSCocoaErrorDomain Code=3840 "Badly formed object around character 122." UserInfo={NSDebugDescription=Badly formed object around character 122.}

I am not totally sure if the problem is in your library.

Thank you! :)

**Comment by tidwall** [comment]
Thank you for finding this issue. I just pushed a fix to the `swift/3.0` branch.

**Comment by segura2010** [comment]
Thank you for you fast fix! :)
